tests/cli/test_package.py

```diff
@@ -186,6 +186,42 @@ def test_cannot_add_tarball_object_without_required_options(self):
             result = self.runner.invoke(add_object_command, cmd)
             self.assertEqual(result.exit_code, 2)
 
+    def test_add_duplicate_object(self):
+        try:
+            h1 = NamedTemporaryFile()
+            h2 = NamedTemporaryFile()
+            dummy_fn_1 = h1.name
+            dummy_fn_2 = h2.name
+        except:
+            return None
```

Comment:
Is this intended to be ignored?

Reply:
In case the files can't be created it would give a false negative, so it'd  be better to just return.

Reply:
Now I'm thinking the correct return value is actually `True`